SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -438,6 +438,11 @@ static FSubobjectToOffsetMap CreateOffsetMapFromActor(USpatialPackageMapClient*
 		// server- and client-side components added in the level.
 		TArray<UActorComponent*> ActorInstanceComponents = Actor->GetInstanceComponents();
 
+		// In non-editor builds, editor-only components can be allocated a slot in the array, but left as nullptrs.
+		ActorInstanceComponents.RemoveAll([](UActorComponent* Component) {
+			return Component == nullptr;
+		});
+
```

Comment:
Nit - this will avoid an extra copy:
```suggestion
		TArray<UActorComponent*> ActorInstanceComponents;

		// In non-editor builds, editor-only components can be allocated a slot in the array, but left as nullptrs.
		Algo::CopyIf(Actor->GetInstanceComponents(), ActorInstanceComponents, [](UActorComponent* Component) {
			return IsValid(Component);
		});
```
Might also need a `#include "Algo/Copy.h"` at the top.